src/test/java/net/sf/jabref/logic/layout/LayoutTest.java

```diff
@@ -86,15 +82,6 @@ public void testHTMLChar() throws IOException {
     }
 
     @Test
-    @Ignore
```

Comment:
Why not adapting it to the current behavior? I think, we drop `<br>` now, which is fine. Why not leaving the additional test?